fawkes/algorithms/algo.py

```diff
@@ -23,6 +23,55 @@
 from fawkes.review.review import Review
 from fawkes.cli.fawkes_actions import FawkesActions
 
+from sentence_transformers import SentenceTransformer
+import numpy as np
+from sklearn.cluster import KMeans
+from gensim.summarization.summarizer import summarize
+
+
+'''
+        @param{list<string>}: sentences - List of review sentence
+        @param{Integer}: num_clusters - User defined number of clusters
+        @returns{list<list<string>}: clustered_sentences - List of clusters of sentences 
+        
+        Groups similar sentences into clusters
+'''
+#Reference : https://github.com/UKPLab/sentence-transformers
+def k_means_classification(sentences, num_clusters):
```

Comment:
Why can't all this logic be put in the summarisation file ?

Provide a clean API for the algorithm to use.

Reply:
These are plugable algorithms, these might change based on performance. Why should the main code to summarise have these ?

Reply:
Then these should be in a separate module. The algo.py should not be polluted with these code.